nikola/plugins/shortcode/emoji/data/LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 -2017 Shayan Rais
```

Comment:
Add a link to the [repo](https://github.com/shanraisshan/EmojiCodeSheet) somewhere.